docs/reference/sciencedirect/SerialTitleISSN.rst

```diff
@@ -1,8 +1,8 @@
-pybliometrics.sciencedirect.SerialTitle
-=======================================
+pybliometrics.sciencedirect.SerialTitleISSN
+===========================================
 
-`SerialTitle()` implements the `Serial Title API <https://dev.elsevier.com/documentation/SerialTitleAPI.wadl>`_.  It offers basic information on registered serials (also known as sources), including publisher details, identifiers, and various metrics.
-Note that this class accesses the same API endpoint as the :class:`pybliometrics.scopus.SerialTitle` class.
+`SerialTitleISSN()` implements the `Serial Title API <https://dev.elsevier.com/documentation/SerialTitleAPI.wadl>`_.  It offers basic information on registered serials (also known as sources), including publisher details, identifiers, and various metrics.
+Note that this class accesses the same API endpoint as the :class:`pybliometrics.scopus.SerialTitleISSN` class.
```

Comment:
I think this is wrong. Let's drop it. We can include a note specifying the difference to the other class for the same API though.